check50/__main__.py

```diff
@@ -266,8 +266,14 @@ def main():
                         metavar="FILE",
                         help=_("file to write output to"))
     parser.add_argument("-v", "--verbose",
-                        action="store_true",
-                        help=_("display the full tracebacks of any errors (also implies --log)"))
+                        action="store",
+                        nargs="?",
+                        default="none",
```

Comment:
Why not make the default something falsey (None/False) instead? That way things like "if args.verbose" would still work.

Reply:
Agreed.

Reply:
I went with `""` instead of `False`/`None`, that at least keeps the type the same.